_maps_2024/06-jiatong-raster.md

```diff
@@ -0,0 +1,17 @@
+---
+
+day: 6
+creator: Jiatong Su
+title: Day 6 | Raster
+thumbnail: /assets/thumbnails/2024/06-jiatong-raster.png
+source: https://github.com/sujiatong/30DayMap-Jiatong/blob/main/day-6-raster/day-6.Rmd
+
+---
+
+
+This plot illustrates NDVI value in Philadelphia, which utilize Landsat 8 image in October 21, 2024.
+NDVI (Normalized Difference Vegetation Index) measures vegetation health using satellite imagery.
+It ranges from -1 to 1, where:
+  - Values close to 1 indicate healthy, dense vegetation.
+  - Values near 0 represent bare soil.
+  - Negative values indicate water or non-vegetated areas.
```

Comment:
Adding in the image here. The way that the template works is that if you don't have any information after the front matter then the image thumbnail image will be shown by default, but if you do have information after the front matter then you have to explicitly add the thumbnail image if you want it.

```suggestion
  - Negative values indicate water or non-vegetated areas.
![NDVI Plot Image](assets/thumbnails/2024/06-jiatong-raster.png)
```